nimbus/helm/templates/nimbus-deployment.yaml

```diff
@@ -35,6 +35,8 @@ spec:
         - --data-dir={{ $client.dataDirPath }}
         - --validators-dir={{ $client.validatorsDirPath }}
         - --secrets-dir={{ $client.secretsDirPath }}
+        - --metrics-address=0.0.0.0
+        - --metrics:on
```

Comment:
Do we need `:on` ? Althought, [Set up a systemd service](https://nimbus.guide/beacon-node-systemd.html?highlight=metrics#1-create-a-systemd-service) mentions `on`  and `off` options, [cmd line options](https://nimbus.guide/options.html) doesn't mention anything.